Detectors/TPC/workflow/CMakeLists.txt

```diff
@@ -19,13 +20,14 @@ o2_add_library(TPCWorkflow
                        src/TrackReaderSpec.cxx
                        src/RawToDigitsSpec.cxx
                        src/LinkZSToDigitsSpec.cxx
+                       src/ZSSpec.cxx
                TARGETVARNAME targetName
                PUBLIC_LINK_LIBRARIES O2::Framework O2::DataFormatsTPC
                                      O2::DPLUtils O2::TPCReconstruction
                                      O2::TPCCalibration O2::TPCSimulation 
                                      O2::TPCEntropyCoding)
 
-o2_add_executable(reco-workflow
+   o2_add_executable(reco-workflow
                   COMPONENT_NAME tpc
                   SOURCES src/tpc-reco-workflow.cxx
                   PUBLIC_LINK_LIBRARIES O2::TPCWorkflow)
```

Comment:
Please fix indentation (this is a cmake file, so clang format will not take care of)

Reply:
o2_add_library and o2_add_executable are still not aligned.